front/style/pages/_referent.scss

```diff
@@ -1,7 +1,10 @@
 .referent {
+    h2 {
+        font-family: $font-gillsans;
+    }
+
     &__filters {
-        background: $gray--lighter;
-        padding: 10px;
+        border-top: 2px solid #e7e7e7;
```

Comment:
Can you use a color variable available in the colors mixin please ?

https://github.com/EnMarche/en-marche.fr/blob/master/front/style/mixins/_colors.scss

Reply:
I don't find it in colors.scss, anyway I removed border color, no need